e2e/core/test.ts

```diff
@@ -28,4 +30,13 @@ export const test = base.extend<CustomTestFixtures, CustomWorkerFixtures>({
     },
     { scope: 'test', auto: true },
   ],
+
+  visit: [
+    async ({ api, patient }, use) => {
+      const visit = await startVisit(api, patient.uuid);
+      await use(visit);
+      await endVisit(api, visit);
+    },
+    { scope: 'test', auto: true },
+  ],
```

Comment:
https://github.com/openmrs/openmrs-esm-patient-chart/pull/2487#discussion_r2153773372

Reply:
all the current spec test files so far need an active visti and for the next ones in the pipeline that am going to raise

Reply:
hello @jayasanka-sack i did update the PR